lesson26/src/forgotpassword.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
nits
```suggestion
<html lang="ja">
```

Reply:
Thanks for the review!

Changed "en" to "ja".
 1ddd796